### PR TITLE
Windows build

### PR DESCRIPTION
**Pull request by lhofmann**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lhofmann** [comment]
@conda-forge-admin please rerender

**Comment by lhofmann** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -14,7 +14,7 @@ cmake .. -G "Ninja" ^
     -DTTK_ENABLE_CPU_OPTIMIZATION=OFF ^
     -DTTK_ENABLE_KAMIKAZE=OFF ^
     -DTTK_ENABLE_MPI=OFF ^
-    -DTTK_ENABLE_OPENMP=ON
+    -DTTK_ENABLE_OPENMP=OFF
```
Comment (julien-tierny) [inline recipe/bld.bat]: sorry I forgot to mention that we indeed recommend to disable OpenMP on windows (https://topology-tool-kit.github.io/installation-windows.html). ttk makes many OpenMP 2 calls (which are not supported by msvc).

Comment (lhofmann) [reply recipe/bld.bat]: Thanks for your message! I was confused, because it actually builds just fine with vs2019, but not with vs2015.

**Pull request merged**